### PR TITLE
docs: Improvement to main module documentation in regard to gp3 adoption

### PR DESCRIPTION
**Pull request by etiennechabert**
## Description
- Copying docs improvement from submodule to main module regarding `iops`/`storage_throughput`
- Same changes as: https://github.com/terraform-aws-modules/terraform-aws-rds/pull/457

## Motivation and Context
This reminder from @lucasvuotto: https://github.com/terraform-aws-modules/terraform-aws-rds/pull/456#issuecomment-1357881179

## Breaking Changes
Just docs

## How Has This Been Tested?
No tests done

### CODE REVIEW

**Comment by etiennechabert** [comment]
Please do not merge before: https://github.com/terraform-aws-modules/terraform-aws-rds/pull/462

Both PRs are touching IOPS variable, I will merge `master` into this branch after the first PR is merged

**Comment by etiennechabert** [comment]
Sorry @bryantbiggs undid your change... please put it back if you prefer your tittle

**Review by bryantbiggs — APPROVED** [review]

**Comment by antonbabenko** [comment]
This PR is included in version 5.2.3 :tada:

**Pull request merged**
